### DISCUSSION TITLE
Is the config in `overrides` merged into the main config?

### DISCUSSION THREAD

**Discussion by haocheng6** [eslint/eslint#14929] [Q&A] [answered]
I read [How do overrides work?](https://eslint.org/docs/user-guide/configuring/configuration-files#how-do-overrides-work) in the documentation, but it is not clear whether the config in `overrides` completely replaces the regular config or they will get merged into the regular config.

A similar issue was also brought up in the eslint website repository. (See [#504](https://github.com/eslint/website/issues/504).)

Thank you. :)

**Comment by aladdin-add** [comment, marked as answer]
they will be merged.
